include/boost/geometry/strategies/cartesian/turn_in_ring_winding.hpp

```diff
@@ -107,13 +108,12 @@ class turn_in_ring_winding
         return s1y < s2y ? (py >= s1y && py <= s2y) : (py >= s2y && py <= s1y);
     }
 
-    template <typename Point, typename PointOfSegment, typename Ring>
+    template <typename Point, typename PointOfSegment>
     static inline void apply_on_boundary(Point const& point,
                              PointOfSegment const& s1,
                              PointOfSegment const& s2,
                              place_on_ring_type place_on_ring,
-                             counter& the_state,
-                             Ring const& full_ring)
+                             counter& the_state)
```

Comment:
@barendgehrels Is that ok or should I rather leave the parameter and remove the name?

Reply:
It's OK, thanks. I also answered elsewhere before seeing this comment.